lib/kubernetes-deploy/statsd.rb

```diff
@@ -4,23 +4,40 @@
 
 module KubernetesDeploy
   class StatsD
+    extend ::StatsD
+
+    PREFIX = "KubernetesDeploy"
+
     def self.duration(start_time)
       (Time.now.utc - start_time).round(1)
     end
 
     def self.build
-      ::StatsD.default_sample_rate = 1.0
-      ::StatsD.prefix = "KubernetesDeploy"
```

Comment:
There's no point in setting `StatsD.prefix` (_not_ `::StatsD.prefix`) here, since `::StatsD.prefix` is what's used inside the library (hence the need to add the `:prefix` option in the first place). I think leaving it here would be misleading so I'm opting to just be rid of it

Reply:
Can you leave this comment above line 28. Since its important but not obvious